pages/tutorials/excel-sheet-into-python-dashboard.mdx

```diff
@@ -0,0 +1,157 @@
+---
+ogTitle: Turn excel sheet into Python dashboard
+description: How to create an interactive dashboard with Python using data from an Excel sheet in Jupyter Notebook. 
+ogImage: tutorials
+---
+
+import {Screenshot} from 'components/screenshot';
+import excelSheet from "../../public/images/tutorials/dashboards/excel-dashboard/excel-sheet.png";
+
+# Turn excel sheet into Python dashboard
+
+We created a simple Python dashboard using data from an Excel sheet and `pandas`, `mercury` and `altair` packages. 
+In this article you will learn:
+
+- how to load excel sheets to your Jupyter Notebook,
+- how to use Markdown, NumberBox, Range and Table `mercury` widgets,
+- how to create charts using `vega-altair`,
+- how to turn your notebook into Web App.
+
+
+## Import required packages
+
+You will need following packages: 
+
+```python
+pandas 
+mercury
+altair
+```
+
+- `pandas`  is  for data manipulation and analysis.
+- `mercury` will be used for interactive widgets and for running notebook as a web app.
+- `altair` for creating declarative statistical visualizations.
+
+Make sure that you have installed them before start doing anything.
```

Comment:
Maybe add at the end:

Please check articles about [Pandas module not found](https://mljar.com/modulenotfounderror/no-module-named-pandas/) and [Altair module not found](https://mljar.com/modulenotfounderror/no-module-named-altair/) in the case of any problems.